docs/development/dev-services.rst

```diff
@@ -203,9 +203,9 @@ Install dependencies
   sudo pip-2.7 install docutils
   brew install mercurial
 
-Install lessc
+Install node dependencies
 ::
-  sudo npm install -g less
+  npm install less
```

Comment:
Since we now have a ``package.json`` and the install is part of ``bootstrap.sh``, I suspect this can actually be entirely removed. As it stands, it will install it the root directory and not it ``./webapp-django``, so I don't think it will be any help.